x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -65,6 +65,67 @@ export const registerCollector: RegisterCollector = ({
         },
       },
       detectionMetrics: {
+        detection_rules: {
+          detection_rule_usage: {
+            query: {
+              enabled: { type: 'long' },
```

Comment:
Do you mind documenting the fields? You can do so by adding the description like in the example:
```
enabled: { type: 'long', _meta: { description: 'Number of enabled detection rules of type query' } },
```

We are trying to enforce documentation to, at least, the new fields.

Reply:
Thanks! Updated in https://github.com/elastic/kibana/pull/95659/commits/35d94d7d4e7196d28be738f3d403c6d1cacec5a8